client/src/common/model/actions.js

```diff
@@ -0,0 +1,29 @@
+import socket from 'sockets';
+import { CommonActionTypes } from 'common/model/actionTypes';
+
+const clearMetaDataSuccess = view => ({
+  type: CommonActionTypes.LEAVE_VIEW,
+  payload: view
```

Comment:
Consider making the payload always an object to make extension easy when for example, `userId` is needed.